client/src/legacy/LeftAndMain.FieldHelp.js

```diff
@@ -10,28 +10,40 @@ $.entwine('ss', function($) {
    * input fields (e.g. GridField), and aren't compatible
    * with showing tooltips.
    */
-  $(".cms .field.cms-description-tooltip").entwine({
+  $(".cms .field.cms-description-tooltip .form__field-holder").entwine({
     onmatch: function() {
       this._super();
 
-      var descriptionEl = this.find('.description'), inputEl, tooltipEl;
+      var descriptionEl = this.find('.form__field-description');
       if(descriptionEl.length) {
+        var offset = this.css('padding-left');
         this
           // TODO Remove title setting, shouldn't be necessary
           .attr('title', descriptionEl.text())
-          .tooltip({content: descriptionEl.html()});
+          .tooltip({
+            content: descriptionEl.html(),
+            position: {
+              my: "left+" + offset + " top"
```

Comment:
Default is `left top+15`, which appears below and to the left of the input - this aligns it against the input as per screenshot above

Reply:
Couldn't we had a class to make it visible instead so it's managed in CSS rather than JS? Maybe not a big deal but I'd probably be using `.is-visible` on the `.form__field-description` elements...